docs/app/Examples/elements/Icon/IconSet/IconSetExamples.js

```diff
@@ -20,17 +23,25 @@ ${content}
 )
 
 const categorySrc = (icons) => (
-  template(_.map(icons, icon => (`          <Icon name='${icon}' />`).join('\n')))
+  template(_.map(icons, icon => (`          <Icon name='${icon}' />`)).join('\n'))
 )
 
 export default class IconSetExamples extends Component {
   render() {
     return (
       <ExampleSection title='Icon Set'>
+        An icon set contains an arbitrary number of glyphs
+        <Message className='warning'>
+          Icons serve a very similar function to text in a page. In Semantic icons receive a special tag &lt;i&gt; which allow for an abbreviated markup when sitting along-side text.
+        </Message>
+        <Message>
+          Semantic includes a complete port of <a href='http://fortawesome.github.io/Font-Awesome/' target='_blank'>Font Awesome 4.5.6</a> designed by <a href='http://www.twitter.com/davegandy' target='_blank'>Dave Gandy</a> for its standard icon set.
+        </Message>
+        <br />
         {_.map(categories, category => (
           <ComponentExample
             title={category.name}
-            description='Icons can represent types of content found on websites'
+            description={category.description}
```

Comment:
Hm.  Since the utils are required by the library, they are also bundled with users' apps.  I'd prefer to keep the docs completely separate from the library source so it doesn't end up in application source.

Any ideas how we can keep the docs out of the source code while still keeping this loop?